src/screens/Order/Orders.res

```diff
@@ -53,13 +53,42 @@ let make = (~previewOnly=false) => {
             ]->getJsonFromArrayOfJson,
           )
         }
-
+        let obj = Dict.make()
+        obj->Dict.set(
+          "start_amount",
+          switch dict->Dict.get("amount_filter.start_amount") {
+          | Some(value) =>
+            value
+            ->JSON.Decode.string
+            ->Option.getOr("")
+            ->Float.fromString
+            ->Option.getOr(0.0)
+            ->JSON.Encode.float
+          | None => JSON.Encode.null
+          },
+        )
+        obj->Dict.set(
+          "end_amount",
+          switch dict->Dict.get("amount_filter.end_amount") {
+          | Some(value) =>
+            value
+            ->JSON.Decode.string
+            ->Option.getOr("")
+            ->Float.fromString
+            ->Option.getOr(0.0)
+            ->JSON.Encode.float
+          | None => JSON.Encode.null
+          },
+        )
         dict
         ->Dict.toArray
         ->Array.forEach(item => {
           let (key, value) = item
           filters->Dict.set(key, value)
         })
+        filters->Dict.set("amount_filter", obj->JSON.Encode.object)
```

Comment:
maybe have a function whichtakes array of keys and travelrs thorugh each string and delete the keys